generators/chipyard/src/main/scala/ChipTop.scala

```diff
@@ -23,12 +23,10 @@ case object BuildSystem extends Field[Parameters => LazyModule]((p: Parameters)
  * drive clock and reset generation
  */
 
-class ChipTop(implicit p: Parameters) extends LazyModule with HasTestHarnessFunctions {
-  // A publicly accessible list of IO cells (useful for a floorplanning tool, for example)
-  val iocells = ArrayBuffer.empty[IOCell]
-
+class ChipTop(implicit p: Parameters) extends LazyModule
```

Comment:
Can this extend `BindingScope` so that we can use the lazy binders to attach different resources.

Reply:
Can you do this in your FPGA PR?

Reply:
Sure. I figured this would be innocuous here but I can move it to the other PR.